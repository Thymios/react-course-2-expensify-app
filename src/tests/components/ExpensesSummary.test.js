import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import expenses from '../fixtures/expenses';

test('should correctly render ExpensesSummary with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={1415} />
    );
    expect(wrapper).toMatchSnapshot();
});

test('should render correctly ExpensesSummary with multiple expenses', () => {
    const wrapper =  shallow(<ExpensesSummary expenseCount={3} expensesTotal={141567} />);
    expect(wrapper).toMatchSnapshot();
});

