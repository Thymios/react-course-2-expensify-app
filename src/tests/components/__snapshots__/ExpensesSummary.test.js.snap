// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $14.15
  </h1>
</div>
`;

exports[`should render correctly ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    3
     
    expenses
     totalling 
    $1,415.67
  </h1>
</div>
`;
